src/displayapp/screens/Weather.cpp

```diff
@@ -0,0 +1,203 @@
+#include "displayapp/screens/Weather.h"
+#include <lvgl/lvgl.h>
+#include "components/ble/SimpleWeatherService.h"
+#include "components/settings/Settings.h"
+#include "displayapp/DisplayApp.h"
+#include "displayapp/screens/WeatherSymbols.h"
+#include "displayapp/InfiniTimeTheme.h"
+
+using namespace Pinetime::Applications::Screens;
+
+namespace {
+  lv_color_t temperatureColor(int16_t temperature) {
```

Comment:
Please use PascalCase for function names

Reply:
Fixed